lib/systeminfo/systeminfo.go

```diff
@@ -238,6 +238,10 @@ func collectFilesystemUsage(fs []storage.Filesystem) (result storage.FilesystemS
 	for _, mount := range fs {
 		usage := sigar.FileSystemUsage{}
 		if err := usage.Get(mount.DirName); err != nil {
+			if os.IsNotExist(err) {
```

Comment:
Just thinking, maybe it would be better to treat any error as non-critical here and just log it. In case it fails with like a permission error or something else.

Reply:
Yeah, maybe you're right.